app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,64 @@
+module Idv
+  class HowToVerifyController < ApplicationController
+    include IdvStepConcern
+    include RenderConditionConcern
+
+    before_action :confirm_step_allowed
+
+    check_or_render_not_found -> { self.class.enabled? }
+
+    def show
+      analytics.idv_doc_auth_how_to_verify_visited(**analytics_arguments)
+      @idv_how_to_verify_form = Idv::HowToVerifyForm.new
+    end
+
+    def update
+      result = Idv::HowToVerifyForm.new.submit(how_to_verify_form_params)
+
+      analytics.idv_doc_auth_how_to_verify_submitted(
+        **analytics_arguments.merge(result.to_h),
+      )
+
+      if result.success?
+        if how_to_verify_form_params['selection'] == Idv::HowToVerifyForm::REMOTE
+          redirect_to idv_document_capture_url
```

Comment:
Was it deliberate to make it redirect to the document capture page if the user picks remote proofing? The [previous behavior](https://github.com/18F/identity-idp/pull/9452/files#diff-97ffc12bfb8463bc3769768f7253d0be31f903f506767429d28fed58bd04ef10L35) was to route to the hybrid handoff page -- I think we should keep that unless this was on purpose

```suggestion
          redirect_to idv_hybrid_handoff_url
```